core/src/components/range/range.ios.scss

```diff
@@ -28,11 +27,11 @@
 }
 
 ::slotted([slot="start"]) {
-  @include margin(0, var(--margin), 0, 0);
+  @include margin(0, 16px, 0, 0);
 }
 
 ::slotted([slot="end"]) {
-  @include margin(0, 0, 0, var(--margin));
```

Comment:
I incorrectly assumed that the margins in the slots should be the same as the margin between the control and the label, so I reverted the `--margin` usage.